susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -36,6 +36,9 @@ mgr_server_localhost_alias_absent:
 {%- elif salt['file.file_exists']('/etc/alinux-release') %}
 {% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/alibaba/' ~ grains['osmajorrelease'] ~ '/bootstrap/' %}
 
+{%- elif salt['file.file_exists']('/etc/rocky-release') %}
+{% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/rocky/' ~ grains['osmajorrelease'] ~ '/bootstrap/' %}
```

Comment:
I think the path to the bootstrap repo is wrong.

In spacewalk/certs-tools/rhn_bootstrap_strings.py you set `BASE` to "rockylinux" which match the path in "susemanager/src/mgr_bootstrap_data.py " where you define the bootstrap repo at: `/srv/www/htdocs/pub/repositories/rockylinux/8/bootstrap/` . 
Just "rocky" is not enough here.

```suggestion
{% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/rockylinux/' ~ grains['osmajorrelease'] ~ '/bootstrap/' %}
```